src/AuthTemplate.vue

```diff
@@ -6,7 +6,7 @@ import LoadingView from '@/components/Utilities/LoadingView.vue'
 <template>
   <div class="flex flex-col h-screen w-full">
     <NavBar />
-    <div class="mx-3 grow flex">
+    <div class="grow m-6 flex">
```

Comment:
same here, double grow flex in here and router view

Reply:
this one is necessary to keep the router view centered